action.yml

```
@@ -15,6 +15,12 @@ inputs:
 
       Example: `formatters codeqa`.
     default: ''
+  extra-args:
+    description: |-
+      Extra arguments to be passed to nox sessions. If empty, no extra arguments are passed.
+      Can be a space-separated list of arguments
+
+      Example: `--check`
```

Just to make it consistent with the other options :)
```suggestion
      Example: `--check`.
```